app/Http/Controllers/ExpenseController.php

```diff
@@ -581,4 +583,9 @@ public function upload(UploadExpenseRequest $request, Expense $expense)
 
         return $this->itemResponse($expense->fresh());
     }
+
+    public function edocument(EDocumentRequest $request): string
```

Comment:
please considder using the document instead of the full request (UploadedFile).
f.ex. the InboundMailEngine is not able to provide the full request.